src/vs/workbench/workbench.web.api.ts

```diff
@@ -412,6 +412,12 @@ interface IWorkbenchConstructionOptions {
 	 */
 	readonly codeExchangeProxyEndpoints?: { [providerId: string]: string }
 
+	/**
+	 * [TEMPORARY]: This will be removed soon.
+	 * Endpoints to be used for proxying repository tarball download calls in the browser.
+	 */
+	readonly tarballProxyEndpoints?: { [providerId: string]: string }
```

Comment:
Please create a follow up issue to keep in mind to remove this API again.

Reply:
Maybe we also prefix with an `_`?

Reply:
@bpasero filed https://github.com/microsoft/vscode/issues/138335. Should I include that in the comment too?

@joaomoreno is that naming convention done for internally-consumed commands, or for commands that are proposed and subject to removal?

Reply:
No I think a back reference is not needed.